cachematrix.R

```diff
@@ -1,15 +1,15 @@
-## Put comments here that give an overall description of what your
-## functions do
-
-## Write a short comment describing this function
-
-makeCacheMatrix <- function(x = matrix()) {
-
-}
-
-
-## Write a short comment describing this function
-
-cacheSolve <- function(x, ...) {
-        ## Return a matrix that is the inverse of 'x'
-}
+## Put comments here that give an overall description of what your

```

Comment:
this function computes the inverse of the special "matrix " returned by makeCacheMatrix